packages/realm-react/src/AppProvider.tsx

```diff
@@ -18,11 +18,43 @@
 import { isEqual } from "lodash";
 import React, { createContext, useContext, useLayoutEffect, useRef, useState } from "react";
 import Realm from "realm";
+import { AuthOperationName, AuthResult, OperationState } from "./types";
+
+type AppContextValue = {
+  app: Realm.App | null;
+};
```

Comment:
Any reason for packing this in an object now that it's a single property again?

```suggestion
type AppContextValue = Realm.App | null;
```